addon/components/fixtable-footer.js

```diff
@@ -3,7 +3,6 @@ import layout from '../templates/components/fixtable-footer';
 
 export default Ember.Component.extend({
   layout,
-  classNames: ['fixtable-footer'],
```

Comment:
is this going to be a breaking change?

Reply:
This class moves from being directly on the component to being applied to a new div that wraps the component, so it shouldn't be a problem, and hasn't been an issue in testing this linked against our Ember engine.

Reply:
Cool, sounds good.